src/vivarium/framework/components/manager.py

```diff
@@ -116,27 +116,52 @@ def add_components(self, components: Union[List, Tuple]):
         """
         self._add_components(self._components, components)
 
-    def _add_components(self, component_list: OrderedComponentSet, components: Union[List, Tuple]):
+    def _add_components(self, component_list: OrderedComponentSet, components: Union[List[Any], Tuple[Any]]):
         for component in components:
             if isinstance(component, list) or isinstance(component, tuple):
                 self._add_components(component_list, component)
             else:
                 component_list.add(component)
 
-    def get_components(self, component_type: Any) -> List:
-        """Gets a list of components currently held by the component manager
-        that are instances of a specific type. Does not include managers.
+    def get_components_by_type(self, component_type: Any) -> List[Any]:
```

Comment:
component type can actually be `type` here

Reply:
this whole time I didn't realize you meant lowercase type :man_facepalming: